api/pkg/apis/v1alpha1/providers/states/k8s/k8s.go

```diff
@@ -411,13 +411,14 @@ func (s *K8sStateProvider) Get(ctx context.Context, request states.GetRequest) (
 }
 
 // Implmeement the IConfigProvider interface
-func (s *K8sStateProvider) Read(object string, field string) (string, error) {
+func (s *K8sStateProvider) Read(object string, field string, scope string) (string, error) {
```

Comment:
I realize that neither the original implementation nor the updated implementation implemented IConfigProvider interface. We need to make sure this is updated to comply with interface and add test cases for it.

Reply:
Correct. Sounds good.